Jenkinsfile

```diff
@@ -105,6 +105,7 @@ pipeline {
                 checkout scm
                 dir("$WORKSPACE/docker") {
                     script {
+                        MavenRun('package')
```

Comment:
```suggestion
```

Reply:
I think that the maven command should be executed in the `$WORKSPACE` folder, not in `$WORKSPACE/docker`

Reply:
Indeed the DockerBuild method accepts a 3rd argument for the path of the Dockerfile, thanks for spotting this! @alberto-brigandi

Reply:
You're right @orviz, package should be enough